SmartDeviceLink/SDLRPCNotificationNotification.m

```diff
@@ -31,6 +31,11 @@ - (__kindof SDLRPCNotification *)notification {
     return _userInfo[SDLNotificationUserInfoObject];
 }
 
+- (BOOL)isNotificationKindOfClass:(Class)aClass {
+    NSAssert([self.notification isKindOfClass:aClass], @"A notification was sent with an unanticipated object");
```

Comment:
Both of these now exist. I'm not keen on this but it is for parity between responses and notifications.